CHARTER.adoc

```diff
@@ -31,11 +36,17 @@ The TODO Group shall be composed of the General Members and a Steering Committee
  ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members shall appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
  ... If any committee member should resign or otherwise be unable to complete their term for any reason, with six (6) months or fewer remaining in their term, their position will be left unfilled until the regularly scheduled election or appointment process for that position.
  ... Elections shall be run using the Condorcet-IRV method through the Cornell online service (+++<u>+++http://civs.cs.cornell.edu/+++</u>+++). The methodology or service used in future elections may be changed via a five (5) member approval vote of current members of the committee at that time.
+ .. Removal:
+ ... Steering Committee members shall qualify for removal for:
+ .... Violation of the TODO Group Code of Conduct.
+ .... Inexcused absence from two (2) consecutive Steering Committee meetings.
+ .... Regular non-participation in Steering Committee votes and activities via email, real-time chat, or in person.
```

Comment:
Maybe we can be more specific about what "regular" means here? for instance, *being inactive longer than three months* (no email responses, rea-time chat participation, monthly calls, etc)

Reply:
100% agree! We definitely need to define that.

I advise that the "regular non-participation" limit be shorter than 3 months. That's 25% of a year when someone may have ghosted w/o the SC having recourse for taking action. 6-8 weeks would be my preference.

My expectation here is that along with these charter changes there'd be policies defined for how to go about some of these things. Policies/procedures are easier to evolve than the charter, after all. A policy/procedure for this non-participation thing could be something like "step 1: reach out the the SC member to see what's up; step 2: if no reply in $timeframe, SC shall draft and vote on a proposal to remove the member from the SC", etc.

Reply:
I agree. How about 8 weeks, as this is somewhat aligned with missing 2 SC meetings?

```suggestion
 .... Non-participation in Steering Committee votes and activities via email, real-time chat, or in person for more than consecutive 8 weeks.
```

Reply:
I support the eight week non-participation change, but would like us to consider whether we change the charter to allow for a new nominee to come from the member organization for the non-participating SC member.

In other words, if I am inactive for 8 weeks for e.g. a medical issue, but Red Hat would like to appoint someone to cover meetings in my stead, I would like to make this possible. I believe the replacement should be subject to SC approval. 

If no replacement is recommended, we would then move to working with the current charter guidelines - replacement by appointment of SC in first six months of term; no replacement required in last six months of term.